### PR TITLE
add Rust as dependencies for cryptography, add XLS support for data d…

### PR DESCRIPTION
**Pull request by linh-ngophuochoai**
Fix dependency issue for cryptography due to Rust
Add XLS support for Data Driver

### CODE REVIEW

**Comment by UltimateDogg** [comment]
Can you leave the existing whitesapce?

**Comment by linh-ngophuochoai** [comment]
Hi, I am not so sure what does it mean by whitespace. I believe it related to CRLF as I am using Windows machine. Converted to LF and add some comment. Hope it helps

**Review by ppodgorsek — CHANGES_REQUESTED** [review]
As @UltimateDogg pointed out, there was no need to change so many lines, whitespace must be consistent.
Please see for yourself in the "Files changed" tab at the top of this PR.

File: Dockerfile [hunk 1]
```diff
@@ -59,63 +59,70 @@ COPY bin/run-tests-in-virtual-screen.sh /opt/robotframework/bin/
...
+  # Continue with system dependencies
+  gcc \
+  g++ \
+  libffi-dev \
+  linux-headers \
+  make \
+  musl-dev \
+  openssl-dev \
+  which \
+  wget \
```
Comment (ppodgorsek) [inline Dockerfile:76]:
```suggestion

    # Install dependencies for cryptography due to https://github.com/pyca/cryptography/issues/5771
    cargo \
    rust \

    # Continue with system dependencies
    gcc \
    g++ \
    libffi-dev \
    linux-headers \
    make \
    musl-dev \
    openssl-dev \
    which \
    wget \
```

**Review by ppodgorsek — CHANGES_REQUESTED** [review]

File: Dockerfile [hunk 2]
```diff
@@ -115,7 +121,7 @@ RUN apk update \
     && mv geckodriver /opt/robotframework/drivers/geckodriver \
     && rm geckodriver-$GECKO_DRIVER_VERSION-linux64.tar.gz \
 
-# Clean up buildtime dependencies
+  # Clean up buildtime dependencies
```
Comment (ppodgorsek) [inline Dockerfile]:
```suggestion
# Clean up buildtime dependencies
```

**Review by UltimateDogg — APPROVED** [review]

**Review by ppodgorsek — APPROVED** [review]

**Pull request merged**